assets/js/base/components/button/style.scss

```diff
@@ -79,17 +85,6 @@ body:not(.woocommerce-block-theme-has-button-styles) .wc-block-components-button
 	}
 
 	&.contained {
-		background-color: $gray-900;
-		color: $white;
-
-		&:disabled,
-		&:hover,
-		&:focus,
-		&:active {
-			background-color: $gray-900;
-			color: $white;
-		}
-
 		&:hover {
 			opacity: 0.9;
```

Comment:
We should also remove this and keep it consistent with core button blocks. We need to double-check the existing styles to ensure we're not overriding any style like this. 

Slack: p1680026511680059/1680024656.322559-slack-C02UBB1EPEF

Reply:
Similar to the hover state before, I also removed this one here.